themes/default/content/docs/guides/self-hosted/ecs-hosted.md

```diff
@@ -50,13 +50,13 @@ To this end, you need to set up the following:
 
 ### Deployment Steps
 
-See the [README](https://github.com/pulumi/pulumi-self-hosted-installers/ecs-hosted/README.md) file provided with the installer package for detailed deployment steps.
+See the [README](https://github.com/pulumi/pulumi-self-hosted-installers/tree/master/ecs-hosted/README.md) file provided with the installer package for detailed deployment steps.
 
 ## ECS-Hosted System Management and Maintenance
 
 ### Pulumi Service Updates
 
-When deploying the service, it is recommended to pin the Pulumi Service image tag to a specific version. See the installer's [README](https://github.com/pulumi/pulumi-self-hosted-installers/ecs-hosted/README.md) file to set the `imageTag` configuration property for the installer to use.
+When deploying the service, it is recommended to pin the Pulumi Service image tag to a specific version. See the installer's [README](https://github.com/pulumi/pulumi-self-hosted-installers/tree/master/ecs-hosted/README.md) file to set the `imageTag` configuration property for the installer to use.
```

Comment:
Also here -- same link, just wanted to call it out so it didn't get missed.

Reply:
Yes I have a PR for the self-hosted-installers repo to change the file name to README.md from readme.md. So it'll work once that PR is done.